packages/editor/src/components/block-list/style.scss

```diff
@@ -346,14 +346,16 @@
 		width: 100%;
 
 		// When images are floated, the block itself should collapse to zero height.
-		margin-bottom: 0;
 		height: 0;
 
 		// Hide block outline when an image is floated.
 		.editor-block-list__block-edit {
 			&::before {
 				content: none;
 			}
+
+			// This margin won't collapse on its own, so zero it out.
```

Comment:
I'm not *quite* sure what this means in this context, so maybe a bit more commenting would be handy.